config/routes.rb

```diff
@@ -0,0 +1,13 @@
+Rails.application.routes.draw do
+  root to: "homepages#index"
+
+  get "/login", to: "users#login_form", as: "login"
+  post "/login", to: "users#login"
+  post "/logout", to: "users#logout", as: "logout"
+  get "/users/current", to: "users#current", as: "current_user"
+
+  resources :votes, only: [:create]
```

Comment:
I would suggest nesting this route under works, so that you have the work_id in the route.